.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+- id: chartpress
+  name: chartpress --reset
+  files: (Chart|chartpress|values).ya?ml
+  description: Run `chartpress --reset` to clean up helm charts before committing.
+  entry: chartpress --reset
```

Comment:
I considered this but figured it wouldnt work well as chartpress must be executed next to the chartprwss.yaml file, but binserhub and other git repos doesnt have it in the root folder.

Perhaps a param will be required for where one chartpress.yaml file is found.